packages/ovm/src/contracts/ExecutionManager.sol

```diff
@@ -118,52 +117,37 @@ contract ExecutionManager is FullStateManager {
      *       [callBytes (bytes (variable length))]
      * returndata: [variable-length bytes returned from call]
      */
-    function executeCall() external {
+    function executeTransactionRaw() external {
```

Comment:
Would consider massaging the `@notice` above here for clarity as it currently says "Execute a call" and we are renaming the function from `executeCall`